core/src/alpenglow_consensus/block_creation_loop.rs

```diff
@@ -89,6 +90,37 @@ pub struct LeaderWindowNotifier {
     pub window_notification: Condvar,
 }
 
+#[derive(Default)]
+struct BlockCreationLoopTiming {
+    last_submit: u64,
```

Comment:
https://github.com/anza-xyz/agave/blob/5c9878b6bba2d781a15e2ff068ce38885f3ff343/core/src/banking_stage.rs#L98 I think `AtomicInterval` is the more canonical way to do this now

Reply:
The metrics server expects signed types, so we need to convert to `i64`